GamingSupervisor/GamingSupervisor/GameStateIntegration.cs

```diff
@@ -165,7 +165,7 @@ public string SteamID
         }
 
         private readonly object gameTimeLock = new object();
-        private int gameTime;
+        public int gameTime;
```

Comment:
This should not be made public. There is a thread safe getter right below this called "GameTime".

Reply:
I will change that back.